Update intro_inventory page (#691)
ansible/ansible-documentation: docs/docsite/rst/inventory_guide/intro_inventory.rst

The simplest inventory is a single file with a list of hosts and groups. The default location for this file is ``/etc/ansible/hosts``.
You can specify a different inventory file at the command line using the ``-i <path>`` option or in configuration using ``inventory``.

> Ansible :ref:`inventory_plugins` supports a range of formats and sources to make your inventory flexible and customizable. As your inventory expands, you may need more than a single file to organize your hosts and groups. Here are three options beyond the ``/etc/ansible/hosts`` file:

Reviewer: I'm not sure this change is correct. "Ansible inventory plugins support" is correct use of third person plural form.

Hosts in multiple groups
------------------------

> You can (and probably will) put each host in more than one group. For example, a production webserver in a datacenter in Atlanta might be included in groups called ``[prod]`` and ``[atlanta]`` and ``[webservers]``. You can create groups that track:

Reviewer: ```suggestion
You can put each host in more than one group. For example, a production web server in a data center in Atlanta might be included in groups called ``[prod]`` and ``[atlanta]`` and ``[webservers]``. You can create groups that track:
```

Reviewer: I guess it's kind of scope creep but I'd suggest we make these changes while we're in here.

        hosts:
          www[01:50:2].example.com:

> The example above would make the subdomains www01, www03, www05, ..., and www49 match, but not www00, www02, www50 and so on because the stride (increment) is 2 units each step.

Reviewer: I think this actually changes meaning slightly and should be reverted.

Assigning a variable to one machine: host variables
===================================================

> You can easily assign a variable to a single host, and then use it later in playbooks. You can do this directly in your inventory file.

Reviewer: ```suggestion
You can easily assign a variable to a single host and then use it later in playbooks. You can do this directly in your inventory file.
```

Connecting to hosts: behavioral inventory parameters
====================================================

> As described above, setting the following variables controls how Ansible interacts with remote hosts.

Reviewer: "variables control"

ansible_ssh_pipelining
    Determines whether or not to use SSH pipelining. This can override the ``pipelining`` setting in :file:`ansible.cfg`.
ansible_ssh_executable (added in version 2.2)

>     This setting overrides the default behavior to use the system :command:`ssh`. This can override the ``ssh_executable`` setting in :file:`ansible.cfg` under ``ssh_connection`` section.

Reviewer: ```suggestion
    This setting overrides the default behavior to use the system :command:`ssh`. This can override the ``ssh_executable`` setting in :file:`ansible.cfg` under ``ssh_connection``.
```

ansible_become_method
    Allows to set privilege escalation method
ansible_become_user
    Equivalent to ``ansible_sudo_user`` or ``ansible_su_user``, allows you to set the user you become through privilege escalation
ansible_become_password
    Equivalent to ``ansible_sudo_password`` or ``ansible_su_password``, allows you to set the privilege escalation password (never store this variable in plain text; always use a vault. See :ref:`tip_for_variables_and_vaults`)
ansible_become_exe
    Equivalent to ``ansible_sudo_exe`` or ``ansible_su_exe``, allows you to set the executable for the escalation method selected
ansible_become_flags

>     Equivalent to ``ansible_sudo_flags`` or ``ansible_su_flags``, allows you to set the flags passed to the selected escalation method. This can be also set globally in :file:`ansible.cfg` in the ``become_flags`` option under ``privilege_escalation`` section

Reviewer: ```suggestion
    Equivalent to ``ansible_sudo_flags`` or ``ansible_su_flags``, allows you to set the flags passed to the selected escalation method. This can be also set globally in :file:`ansible.cfg` in the ``become_flags`` option under ``privilege_escalation``.
```
